### PR TITLE
Fix reported bugs

### PR DESCRIPTION
**Pull request by VarS311**
From the reported bugs that showed parsing errors, the following were fixed:
- Count keyword (Added to the allowed keywords)
- 'Product2 product_01 = new Product2();' (Made the identifiers support underscores)
- Order keyword (Added to the allowed keywords)
- Webservice as Modifier

### CODE REVIEW

File: apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/lexer/ApexLexer.java [hunk 1]
```diff
@@ -26,7 +26,7 @@
     /**
      * Stores a pattern to identify a keyword.
      */
-    private static final String IDENTIFIER_PATTERN = "(_{0,2}[a-zA-Z][a-zA-Z0-9]*)+";
+    private static final String IDENTIFIER_PATTERN = "(_{0,2}[a-zA-Z][_a-zA-Z0-9]*)+";
```
Comment (JimmyRojas) [inline apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/lexer/ApexLexer.java:29]: Are you sure this change is correct?

Comment (JimmyRojas) [inline apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/lexer/ApexLexer.java:29]: please evaluate the pattern with
myVar_
myVar__
_myVar
__myVar
_myVar_001
_myVar__001
myVar_001
_my_Var
_my_Var_
_my__Var_
_my_Var_001

Comment (JimmyRojas) [inline apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/lexer/ApexLexer.java:29]: test successful

Comment (VarS311) [inline apex-squid/src/main/java/org/fundacionjala/enforce/sonarqube/apex/lexer/ApexLexer.java:29]: @JimmyRojas for example, this case '_myVar_001' should not pass and indeed it doesn't, but I think the issue is it supports any number of underscores, do you think I should control that somehow?

**Pull request merged**